student.rb

```diff
@@ -1,9 +1,12 @@
-require './person'
+require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(name, age, _parent_permission, classroom)
     super(name, age, parent_permission: true)
     @classroom = classroom
+    @classroom.student << (self)
   end
```

Comment:
- [**Optional**]  Please note that this method only serves as belongs-to (**A student belongs to a classroom**). You can simply have a distinct method like this:


```ruby
      def classroom=(classroom)
           @classroom = classroom
            classroom.students.push(self) unless classroom.students.include?(self)
      end
```